backend/testfiles/execution/cloud/db.dark

```diff
@@ -383,13 +383,21 @@ module Roundtrip =
     fetch (fun p -> p.int != 2) = [ otherSample () ]
     fetch (fun p -> p.int == 200) = []
 
-    fetch (fun p -> p.int8 == (Builtin.Int8.fromInt64 2)) = [ sample () ]
-    fetch (fun p -> p.int8 != (Builtin.Int8.fromInt64 2)) = [ otherSample () ]
-    fetch (fun p -> p.int8 == (Builtin.Int8.fromInt64 100)) = []
+    fetch (fun p -> p.int8 == ((Builtin.Int8.fromInt64 2) |> Builtin.unwrap)) = [ sample
+                                                                                    () ]
 
-    fetch (fun p -> p.uint8 == (Builtin.UInt8.fromInt64 2)) = [ sample () ]
-    fetch (fun p -> p.uint8 != (Builtin.UInt8.fromInt64 2)) = [ otherSample () ]
-    fetch (fun p -> p.uint8 == (Builtin.UInt8.fromInt64 100)) = []
+    fetch (fun p -> p.int8 != ((Builtin.Int8.fromInt64 2) |> Builtin.unwrap)) = [ otherSample
```

Comment:
Was there a reason to use fromInt64 instead of `2y`?

Reply:
Yes, using 2y resulted in this error 
```
The CLR type System.SByte isn't natively supported by Npgsql or your PostgreSQL. To use it with a PostgreSQL composite you need to specify DataTypeName or to map it, please refer to the documentation
```
Will check again when I add int16 and int128